src/GeneralStateTestsFiller/stCreateTest/CreateResultsFiller.yml

```diff
@@ -42,38 +43,34 @@ CreateResults:
           [callType]       (and (shr $ 0 240) 0xFF)
           [constructor]    (and (shr $ 0 232) 0xFF)
 
+          ; The contract code          
+          (def 'contractMacro
+                    (lll
+                       (call 0xFFFF 0x60A7 0 0 0 0 0)
```

Comment:
can you explain a logic around. here.  
a contract creating happens. then a created contract has this call 
as a body (call 0xFFFF 0x60A7 0 0 0 0 0).

Reply:
I added comments to explain it:

I did not want to rely on knowing the address at which the contract will get created, because the algorithm might change in the future (for CREATE, anyway - CREATE2 is supposed to give fixed results). So to know that the created contract has been called successfully it calls the 0x60a7 account, and that account changes its storage in a way that the expect: section can identify.